library/core/Cargo.toml

```diff
@@ -23,6 +23,8 @@ optimize_for_size = []
 # Make `RefCell` store additional debugging information, which is printed out when
 # a borrow error occurs
 debug_refcell = []
+# Make `TypeId` store a reference to the name of the type, so that it can print that name.
+debug_typeid = []
```

Comment:
Question: not a T-libs reviewer, but, is this[^all] documented anywhere? Feels like only 3 people will ever know if these features existing otherwise lol.

[^all]: or rather, I guess the core/std options in general.

Reply:
I entirely agree that that’s a problem, and I’m not aware of any documentation for `std` features. I expect the main way people will discover this feature is by looking at the source code of `TypeId` (which is linked from rustdocs, so is trivial to find).

On the other hand, maybe there *is* documentation for `std` features, in which case this PR certainly should add to it.

Reply:
Maybe std-dev-guide but I think that's a submodule of a repo somewhere else.

Reply:
Yeah, better documentation would be nice. But as it's missing for the other features as well, I don't think this should be done in this PR.